macos/RNCClipboard.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,275 @@
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXFrameworksBuildPhase section */
+
+/* Begin PBXGroup section */
+		0948FC8F24DCC218003F28CC = {
+			isa = PBXGroup;
+			children = (
+				0948FC9924DCC218003F28CC /* Products */,
+				0948FC9B24DCC218003F28CC /* RNCClipboard.h */,
+				0948FC9D24DCC218003F28CC /* RNCClipboard.m */,
+			);
+			sourceTree = "<group>";
+		};
+		0948FC9924DCC218003F28CC /* Products */ = {
+			isa = PBXGroup;
+			children = (
+				0948FC9824DCC218003F28CC /* libRNCClipboard.a */,
+			);
+			name = Products;
+			sourceTree = "<group>";
+		};
+/* End PBXGroup section */
+
+/* Begin PBXHeadersBuildPhase section */
+		0948FC9424DCC218003F28CC /* Headers */ = {
+			isa = PBXHeadersBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				0948FC9C24DCC218003F28CC /* RNCClipboard.h in Headers */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXHeadersBuildPhase section */
+
+/* Begin PBXNativeTarget section */
+		0948FC9724DCC218003F28CC /* RNCClipboard */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 0948FCA124DCC218003F28CC /* Build configuration list for PBXNativeTarget "RNCClipboard" */;
+			buildPhases = (
+				0948FC9424DCC218003F28CC /* Headers */,
+				0948FC9524DCC218003F28CC /* Sources */,
+				0948FC9624DCC218003F28CC /* Frameworks */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = RNCClipboard;
+			productName = RNCClipboard;
+			productReference = 0948FC9824DCC218003F28CC /* libRNCClipboard.a */;
+			productType = "com.apple.product-type.library.static";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		0948FC9024DCC218003F28CC /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				LastUpgradeCheck = 1160;
+				TargetAttributes = {
+					0948FC9724DCC218003F28CC = {
+						CreatedOnToolsVersion = 11.6;
+					};
+				};
+			};
+			buildConfigurationList = 0948FC9324DCC218003F28CC /* Build configuration list for PBXProject "RNCClipboard" */;
+			compatibilityVersion = "Xcode 9.3";
+			developmentRegion = en;
+			hasScannedForEncodings = 0;
+			knownRegions = (
+				en,
+				Base,
+			);
+			mainGroup = 0948FC8F24DCC218003F28CC;
+			productRefGroup = 0948FC9924DCC218003F28CC /* Products */;
+			projectDirPath = "";
+			projectRoot = "";
+			targets = (
+				0948FC9724DCC218003F28CC /* RNCClipboard */,
+			);
+		};
+/* End PBXProject section */
+
+/* Begin PBXSourcesBuildPhase section */
+		0948FC9524DCC218003F28CC /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				0948FC9E24DCC218003F28CC /* RNCClipboard.m in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXSourcesBuildPhase section */
+
+/* Begin XCBuildConfiguration section */
+		0948FC9F24DCC218003F28CC /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
```

Comment:
If this is just a plain old xcodeproj then it shouldn't be too hard to pull out the build settings to xcconfigs and use those instead. Ideally this array would be empty in the pbxproj

Reply:
Can confirm, this was an easy task.